pkg/asset/machines/machineconfig/ipv6.go

```diff
@@ -36,7 +36,7 @@ func ForDualStackAddresses(role string) (*mcfgv1.MachineConfig, error) {
 		},
 		Spec: mcfgv1.MachineConfigSpec{
 			Config:          rawExt,
-			KernelArguments: []string{"ip=dhcp,dhcp6"},
+			KernelArguments: []string{"ip=enp3s0:dhcp,dhcp6"},
```

Comment:
hard coding the interface name might not be ideal

Reply:
if we were going to hardcode it, we should document with a :warning: . And say that if the NIC has to be changed, explain how it can be done (by generating manifests first, or do UPI, etc).

Reply:
This seems like a super hardcore change unless this piece of code runs only on OpenStack where you guarantee naming of the interface. But if this is consumed by >1 platform, then we have no guarantee that they all follow (e.g. depending on the driver of the NIC, naming schema used will be different)